pkg/sdk/go/domains.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/absmach/magistrala/internal/apiutil"
```

Comment:
Add id check for `DomainPermissions`, `ListDomainUsers`, `ListDomainUsers`, `changeDomainStatus`, `changeDomainStatus`, `RemoveUserFromDomain`

Reply:
this check is already handled in the api level for these functions.